### PR TITLE
Dedicated memory overhead clarification

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-7892

### CODE REVIEW

**Review by theodore-hyman — APPROVED** [review]
LGTM thank you

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ Before making any changes to your cluster's configuration, review the requiremen
 
 A single node cluster is only appropriate for single-region application development and testing. For single-region production deployments, we recommend a minimum of 3 nodes. The number of nodes you choose also affects your storage capacity and performance. See the [Example](#dedicated-example) for more information.
 
+Note that {{ site.data.products.dedicated }} clusters use some provisioned RAM for system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster.md:21]:
```suggestion
A {{ site.data.products.dedicated }} cluster uses some of its provisioned RAM for the node's operating system and system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster's workloads.
```

File: src/current/cockroachcloud/create-your-cluster.md [hunk 2]
```diff
@@ -87,6 +87,7 @@ The choice of hardware per node determines the [cost](#step-2-select-the-cloud-p
     ----------|------------
     Transactions per second | Each vCPU can handle around 1000 transactions per second. For example, 2 vCPUs can handle 2000 transactions per second and 4 vCPUs can handle 4000 transactions per second.
     Scaling | When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For most production applications, we recommend **at least 4 to 8 vCPUs per node**.
+    Memory | Some provisioned RAM may be used for system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster.
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-your-cluster.md:90]:
```suggestion
    Memory | Some of a node's provisioned RAM is used for the node's operating system and system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster's workloads.
```

**Review by mdlinville — APPROVED** [review]
WDYT about this light edit?

**Review by mdlinville — APPROVED** [review]

**Pull request merged**